crates/turborepo-lib/src/cli.rs

```diff
@@ -265,12 +265,27 @@ pub enum Command {
         target: LinkTarget,
     },
     /// Generate a new app / package
+    #[clap(aliases = ["g", "gen"])]
     Generate {
```

Comment:
Default commands in clap are a little tricky, so Generate _becomes_ the default command (run) and then the subcommand is optional.